tests/helpers/skipif.py

```diff
@@ -59,14 +60,49 @@ def test_any_func(...):
         return dict(condition=False, reason="Conditions satisfied, going ahead with the test.")
 
     reasons = [rs for cond, rs in zip(conditions, reasons) if cond]
-    return dict(condition=any(conditions), reason=f"Required: [{' + '.join(reasons)}]",)
+    return dict(
+        condition=any(conditions),
+        reason=f"Required: [{' + '.join(reasons)}]",
+    )
+
+
+class LightSkipif(MarkDecorator):
+    """
+    Light SkipIf wrapper for simple marking specific cases, fully compatible with pytest.mark::
+
+        @LightSkipif(min_torch="0.0")
+        def test_wrapper(arg1):
+            assert arg1 > 0.0
+    """
+
+    def __new__(
+        self,
+        *args,
+        min_gpus: int = 0,
+        min_torch: Optional[str] = None,
+        quantization_available: bool = False,
+        **kwargs
+    ):
+        custom = skipif_args(
```

Comment:
What's the point of keeping the old one?

Reply:
I would rather have the logic separate, but except that there is no extra point...